### DISCUSSION TITLE
Pause 30-days trial period

### DISCUSSION THREAD

**Discussion by dejan-vasic** [community/community#69306] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Can I pause my 30-days trial period, now 29-days?

**Comment by Just-Bax** [comment]
Hello, There is a github official [documentaion](https://docs.github.com/ru/billing/managing-billing-for-github-copilot/managing-your-github-copilot-subscription-for-your-personal-account) about managing subscriptions

**Reply by dejan-vasic** [reply]
"Your cancellation will take effect at the end of your trial period." This means that trial period will continue decrease even after I canceled, isn't it?

**Reply by Just-Bax** [reply]
> "Your cancellation will take effect at the end of your trial period." This means that trial period will continue decrease even after I canceled, isn't it?

No, that statement typically means that your cancellation will not take immediate effect but will instead take effect at the end of your trial period. In other words, you will still have access to the trial period benefits until it naturally expires, but your subscription or access will not be renewed or billed for the full service once the trial period ends.

**Reply by dejan-vasic** [reply]
Sorry but I'm not sure if I understand it yet. Can I stop trial period and activate it again somewhere in the future?

**Reply by Just-Bax** [reply]
> Sorry but I'm not sure if I understand it yet. Can I stop trial period and activate it again somewhere in the future?

Yes, of course you can. I'm talking about the fact that you can cancel your subscription now and after subscription you will have the privilege to use copilot, but then you can buy copilot again and use it

**Comment by davecheney** [comment, marked as answer]
@dejan-vasic We cannot pause a trial in progress but if you contact support they will be able to cancel your trial allowing you to reapply in the future.

**Reply by dejan-vasic** [reply]
Where to find their contact because I tried to send mail to support@github.com and no one responded just pointed me to community forum?

**Reply by davecheney** [reply]
https://support.github.com/contact
